buildenv/jenkins/jck_sync

```diff
@@ -0,0 +1,133 @@
+#!groovy
+
+timestamps{
+    stage('Setup') {
+            SPEC = params.PLATFORM
+            if (params.LABEL !="") {
+                LABEL = params.LABEL
+            } else {
+                LABEL = "ci.role.test&&hw.arch.x86&&sw.os.linux"
+            }
+            node(LABEL) {
+                println "SPEC: ${SPEC}"
+                println "LABEL: ${LABEL}"
+                echo "clone the SCM GIT repo"
+                try {
+                    def gitConfig = scm.getUserRemoteConfigs().get(0)
+                    // Adopt windows machines require env here https://github.com/adoptium/aqa-tests/issues/1803
+                    ref_cache = "${env.HOME}/openjdk_cache"
+                    timeout(time: 1, unit: 'HOURS') {
+                        forceCleanWS()
+                    }
```

Comment:
This is not necessary for this task.

Reply:
Removed environment variable. Do we need to remove forceCleanWS()

Reply:
Yes, remove it.

Reply:
Removed forceCleanWS()